api/java/src/main/java/com/qbot/answeringservice/service/LoginService.java

```diff
@@ -31,11 +33,20 @@ public LoginService(LoginRepository loginRepository, UserRepository userReposito
         this.pwService = pwService;
     }
 
+    public Login createLogin(@NonNull final String userName, @NonNull final String password) {
+        return loginRepository.save(new Login(UUID.randomUUID().toString().toString().toString(), userName,
```

Comment:
Do we have enough toString() here I wonder 😆

Reply:
Whoops. That's what happens when I have a fight with my IDE over text replacement